CHANGELOG.md

```diff
@@ -5,6 +5,7 @@ IMPROVEMENTS:
 * Set failurePolicy to Fail for connectInject mutating webhook so that pods fail to schedule when the webhook is offline. This can be controlled via `connectInject.failurePolicy`. [[GH-1024](https://github.com/hashicorp/consul-helm/pull/1024)]
 * Allow setting global.logLevel and global.logJSON and propogate this to all consul-k8s commands. [[GH-980](https://github.com/hashicorp/consul-helm/pull/980)]
 * Allow setting `connectInject.replicas` to control number of replicas of webhook injector. [[GH-1029](https://github.com/hashicorp/consul-helm/pull/1029)]
+* Add the ability to manually specify a k8s secret containing server-cert via the value `server.serverCert.secretName`. Coupled with the autoEncrypt feature, this removes the need for the tls-init job and can be run in GitOps without having to rely on Helm hooks. [[GH-1024](https://github.com/hashicorp/consul-helm/pull/1046)]
```

Comment:
```suggestion
* Add the ability to manually specify a k8s secret containing server-cert via the value `server.serverCert.secretName`. [[GH-1024](https://github.com/hashicorp/consul-helm/pull/1046)]
```
I think we can just shorten this to here, it's probably not necessary for a user to know about whether or not we deploy tls-init, etc, but they can look this up in the git diff.

Thoughts for/against?

Reply:
Yep makes sense! Also if anyone is curious and wants a written description they can always just open the PR itself and see that line.